examples/hello_world/external_dataset/python_hello_world.py

```diff
@@ -0,0 +1,33 @@
+#  Copyright (c) 2017-2018 Uber Technologies, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Minimal example of how to read samples from a dataset generated by `generate_non_petastorm_dataset.py`
+using plain Python"""
+
+from __future__ import print_function
+
+from petastorm import make_batch_reader
+
+
+def python_hello_world(dataset_url='file:///tmp/external_dataset'):
+    # Reading data from the non-Petastorm Parquet via pure Python
+    with make_batch_reader(dataset_url, schema_fields=["id", "value1", "value2"]) as reader:
+        for schema_view in reader:
+            # Batches are returned
+            for i, val1, val2 in zip(schema_view.id, schema_view.value1, schema_view.value2):
```

Comment:
I think it is ok to show that batches are coming out of the reader. That way all samples do the same.